doc/semantics-of-the-publishing-api.md

```diff
@@ -228,6 +241,27 @@ non-renderable (see [**format**](#format)).
 
 ---
 
+### schema_name
+
+Examples: *manual, policy, redirect*
+
+Required: Conditionally
+
+The `schema_name` specifies the data format of the content item as per the
```

Comment:
I would just say that `schema_name` specifies the name of the schema file in the govuk-content-schemas repository.